corehq/apps/api/resources/v0_5.py

```diff
@@ -1376,3 +1378,19 @@ def get_datasource_data(request, config_id, domain):
     query = cursor_based_query_for_datasource(request_params, datasource_adapter)
     data = response_for_cursor_based_pagination(request, query, request_params, datasource_adapter)
     return JsonResponse(data)
+
+
+@require_GET
+@api_auth()
+@requires_privilege_with_fallback(privileges.API_ACCESS)
+@api_throttle
+def get_cca_user_roles(request, domain):
+    if not toggles.SUPERSET_ANALYTICS.enabled(domain):
```

Comment:
You could replace this `if` block with the decorator `@toggles.SUPERSET_ANALYTICS.required_decorator()`.